apps/etterlatte-saksbehandling-ui/client/src/shared/tags/AdressebeskyttelseGraderingTag.tsx

```diff
@@ -0,0 +1,37 @@
+import { AdressebeskyttelseGradering } from '~shared/types/sak'
+import { Tag } from '@navikt/ds-react'
+
+export const AdressebeskyttelseGraderingTag = ({
+  adressebeskyttelse,
+  size,
+}: {
+  adressebeskyttelse: AdressebeskyttelseGradering
+  size?: 'medium' | 'small' | 'xsmall'
+}) => {
+  switch (adressebeskyttelse) {
+    case AdressebeskyttelseGradering.UGRADERT:
+      return (
+        <Tag variant="neutral" size={size}>
+          Ugradert
+        </Tag>
+      )
```

Comment:
Foreslår at vi ikke viser noe her - tror det at noe "dukker opp" når det er noe spesielt gjør det lettere for saksbehandler å se at noe er annerledes